### PR TITLE
[WIP] install ASC Baseline for testing

### PR DESCRIPTION
**Pull request by adilliadil**

### CODE REVIEW

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -68,6 +68,8 @@ if grep -q "fullgpu" <<< "$FEATURE_FLAGS"; then
     ensureGPUDrivers
 fi
 
+installAscBaseline
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:71]: we should hold on the merge of this change until it's clear.

Comment (adilliadil) [reply vhdbuilder/packer/install-dependencies.sh:71]: At this current state it is experimental

File: .pipelines/.vsts-vhd-builder.yaml [hunk 2]
```diff
@@ -30,14 +30,26 @@ variables:
 phases:
   - phase: build_vhd
     queue:
-      name: Hosted Ubuntu 1604
+      name: ace-test
```
Comment (xuto2) [inline .pipelines/.vsts-vhd-builder.yaml:33]: revert.

File: .pipelines/.vsts-vhd-builder.yaml [hunk 3]
```diff
@@ -30,14 +30,26 @@ variables:
...
+        displayName: Download Asc Baseline
+        inputs:
+          command: download
+          vstsFeed: ASMPackages
+          vstsFeedPackage: asc-baseline
+          vstsPackageVersion: '1.0.0-12'
+          downloadDirectory: vhdbuilder/packer
+      - bash: |
+          pwd
+          ls -R
```
Comment (xuto2) [inline .pipelines/.vsts-vhd-builder.yaml:51]: revert after testing is done

File: .pipelines/.vsts-vhd-builder.yaml [hunk 4]
```diff
@@ -30,14 +30,26 @@ variables:
...
           echo "##vso[task.setvariable variable=MODE]$m"
         displayName: Get Build Mode
+      - task: UniversalPackages@0
+        displayName: Download Asc Baseline
+        inputs:
+          command: download
+          vstsFeed: ASMPackages
+          vstsFeedPackage: asc-baseline
+          vstsPackageVersion: '1.0.0-12'
+          downloadDirectory: vhdbuilder/packer
```
Comment (xuto2) [inline .pipelines/.vsts-vhd-builder.yaml:48]: same for weekly build template

**Pull request merged**